### DISCUSSION TITLE
Help trying to get exact time from streaming endpoint

### DISCUSSION THREAD

**Discussion by sgrossiP** [blakeblackshear/frigate#7452] [Ask A Question] [answered]
Hello Everyone. I'm trying to get an exact time to replay a chunk front vod_ts endpoint but even if I pass to it a start and end time in unix time stamp I am always getting more than needed

example: https://192.168.0.28:5001/vod/bahiauno/start/1691780400/end/1691784000/index.m3u8

In this example, I am asking for a one-hour chunk but vlc and other video players says the length is one hour and twenty seconds and the video starts generally 5 to 8 seconds earlier than requested

does anybody have a clue on how to fix this? Does not matter if I need to fork the repo

I tried to change the segmentation constant from 6s to 1s with no success

this happens with multiple cameras, in lan or over the internet and all of them give me different results

thank you for reading me :)

**Comment by NickM-27** [comment, marked as answer]
> In this example, I am asking for a one-hour chunk but vlc and other video players says the length is one hour and twenty seconds and the video starts generally 5 to 8 seconds earlier than requested

The segments do not start exactly when you want, so there will be cases where recording segments includes a bit extra

**Reply by sgrossiP** [reply]
Changing the constant y const.py will do the work? I am not fully understanding how the record process works

**Reply by NickM-27** [reply]
I don't know what you're asking
